.github/workflows/gradle.yml

```diff
@@ -2,37 +2,30 @@
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
 
 name: Java CI with Gradle
-on:
-  push:
-    branches: [ master ]
-    paths:
-      - 'src/**'
-#  pull_request:
-#    paths:
-#      - 'src/**'
+on: [pull_request]
```

Comment:
Change this back.
The current method runs on push only when a file inside of the `src` folder is edited. This prevents the action from running when something like a GitHub file (example: this file) is changed.